xtable-core/src/main/java/org/apache/xtable/iceberg/IcebergConversionSource.java

```diff
@@ -123,7 +123,10 @@ public InternalTable getTable(Snapshot snapshot) {
     return InternalTable.builder()
         .tableFormat(TableFormat.ICEBERG)
         .basePath(iceTable.location())
-        .name(iceTable.name())
+        .name(
```

Comment:
Can you add a quick inline comment to explain that if the table name is not specified, Iceberg will default to the path so we use the name from the source config? It will be helpful for future developers to know the context when they come across this code.